### PR TITLE
Implement bulk remove case button

### PR DESCRIPTION
**Pull request by asankov**

### CODE REVIEW

File: tcms/testruns/static/testruns/js/get.js [hunk 1]
```diff
@@ -52,6 +67,17 @@ $(document).ready(() => {
...
+function selectedExecutions() {
+    const allSelected = $('.test-execution-checkbox:checked')
+
+    if (!allSelected.length) {
+        const warningText = $('#test_run_pk').data('trans-no-executions-selected')
+        alert(warningText)
+        return { hasSelected: false, selection: [] }
+    }
+
+    return { hasSelected: true, selection: allSelected }
```
Comment (asankov) [inline tcms/testruns/static/testruns/js/get.js:79]: we could delete line 76 and make this `return { hasSelected: allSelected.length !== 0, selection: allSelected }`

I don't have any preference

File: tcms/testruns/templates/testruns/get.html [hunk 2]
```diff
@@ -164,6 +164,9 @@ <h2 class="card-pf-title">
                                     <button type="button" class="btn btn-default">{% trans "Add comment" %}</button>
                                     <button type="button" class="btn btn-default add-hyperlink-bulk " data-toggle="modal" data-target="#add-link-modal">{% trans "Add hyperlink" %}</button>
                                 </div>
+                                <div class="form-group">
+                                    <button type="button" class="btn btn-danger remove-case-bulk">{% trans "Remove case" %}</button>
```
Comment (atodorov) [inline tcms/testruns/templates/testruns/get.html:168]: Remove case -> Delete, see backlog

File: tcms/testruns/static/testruns/js/get.js (hunk 1 above)
Comment (atodorov) [reply tcms/testruns/static/testruns/js/get.js]: I'm not seeing this piece of code so either clarify or mark it as resolved if this is not an issue anymore.

**Review by atodorov — CHANGES_REQUESTED** [review]
String needs update, otherwise LGTM.

File: tcms/testruns/templates/testruns/get.html [hunk 3]
```diff
@@ -164,6 +164,9 @@ <h2 class="card-pf-title">
                                     <button type="button" class="btn btn-default">{% trans "Add comment" %}</button>
                                     <button type="button" class="btn btn-default add-hyperlink-bulk " data-toggle="modal" data-target="#add-link-modal">{% trans "Add hyperlink" %}</button>
                                 </div>
+                                <div class="form-group">
+                                    <button type="button" class="btn btn-danger remove-case-bulk">{% trans "Delete case" %}</button>
```
Comment (atodorov) [inline tcms/testruns/templates/testruns/get.html:168]: Delete case -> Delete, see backlog

**Review by atodorov — CHANGES_REQUESTED** [review]

**Review by atodorov — APPROVED** [review]

**Pull request merged**